db/migrations/20221018090303_create_users_table.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+CREATE TABLE users (
+  id INT NOT NULL AUTO_INCREMENT,
+  name VARCHAR(50) NOT NULL,
+  email VARCHAR(100) NOT NULL,
+  password VARBINARY(200) NOT NULL,
+  birthday VARCHAR(50) NOT NULL,
+  phone_number VARCHAR(50) NOT NULL,
+  point DECIMAL(8,2) NOT NULL,
+  created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+  updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (id)
```

Comment:
- `PK` 속성은 따로 줄을 추가하지 않고도 id 컬럼 속성에 바로 지정해줄 수 있습니다.
- created_at 컬럼의 `TIMESTAMP`는 데이터가 생성될 때의 시간을 컬럼의 값으로 설정합니다.
만약 컬럼 속성이 DATE라면 mysql 8버전에서는 NOW() 함수가 적용이 안되니 이 점은 참고해주세요!

```suggestion
  id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
  name VARCHAR(50) NOT NULL,
  email VARCHAR(100) NOT NULL,
  password VARBINARY(200) NOT NULL,
  birthday VARCHAR(50) NOT NULL,
  phone_number VARCHAR(50) NOT NULL,
  point DECIMAL(8,2) NOT NULL,
  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
  updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP
```